docs/cmdline_migration.md

```diff
@@ -73,19 +73,22 @@ You can use the following command-line options in OpenJ9, just as you did in Hot
 
 These HotSpot command-line options have equivalents in OpenJ9 that are not specified in the same way, but perform a related function:
 
-| HotSpot Option          | OpenJ9 Option                                    | Usage                                                            |
-|-------------------------|--------------------------------------------------|------------------------------------------------------------------|                                                                        
-| `-Xcomp`                | [`-Xjit:count=0`](xjit.md#count)**<sup>1</sup>** | `-Xcomp` disables interpreted method invocations.                |
-| `-Xgc`                  | [`-Xgcpolicy`](xgcpolicy.md)**<sup>2</sup>**     | Configuring your garbage collection policy.                      |
-| `-XX:+UseNUMA`          | [`-Xnuma:none`](xnumanone.md)**<sup>3</sup>**    | Controls non-uniform memory architecture (NUMA) awareness.       |
+| HotSpot Option          | OpenJ9 Option                                    | Usage                                                     |
+|-------------------------|--------------------------------------------------|-----------------------------------------------------------|                                                                        
+| `-Xcomp`                | [`-Xjit:count=0`](xjit.md#count)**<sup>1</sup>** | `-Xcomp` disables interpreted method invocations.         |
+| `-Xgc`                  | [`-Xgcpolicy`](xgcpolicy.md)**<sup>2</sup>**     | Configuring your garbage collection policy.               |
+| `-Xlog`                 | [`-Xlog`](xlog.md)**<sup>3</sup>**               | Controls operating system message logging                 |
+| `-XX:+UseNUMA`          | [`-Xnuma:none`](xnumanone.md)**<sup>4</sup>**    | Controls non-uniform memory architecture (NUMA) awareness.|
 
 <i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Notes:**
 
 1. HotSpot uses `-Xcomp` to force compilation of methods on first invocation. However, this option is deprecated. Whilst it can be used for compatibility, using `-Xjit:count=0` is preferred.
 
 2. HotSpot uses `-Xgc` to both select policies and configure them; OpenJ9 uses `-Xgcpolicy` to select policies, reserving `-Xgc` for configuration.
 
-3. In HotSpot, NUMA awareness is turned off by default and is turned on by using the `-XX:+UseNUMA` option. Conversely, the OpenJ9 VM automatically enables NUMA awareness and uses `-Xnuma:none` to turn it *off*.
+3. To avoid confusion with the HotSpot `-Xlog` option, the OpenJ9 `-Xlog` option is superseded by the [`-Xsyslog`](xsyslog.md) option.
```

Comment:
I'm thinking that most people won't know or care there used to be an OpenJ9 -Xlog option that did something different. Perhaps "the OpenJ9 `-Xlog` option" should be quantified with "prior to the OpenJ9 0.24 release".

Reply:
OK. Will qualify.